kansha/board/comp.py

```diff
@@ -175,9 +151,42 @@ def get_id_by_uri(cls, uri):
     def exists(cls, **kw):
         return DataBoard.exists(**kw)
 
-    def reload_save_template_comp(self):
-        self.save_template_comp = component.Component(SaveTemplateTask(self))
-        self.save_template_comp.on_answer(lambda v: self.reload_save_template_comp())
+    # Main menu actions
+    def add_list(self):
+        new_column_editor = column.NewColumnEditor(len(self.columns))
+        answer = self.modal.call(popin.Modal(new_column_editor))
+        if answer:
+            index, title, nb_cards = answer
+            self.create_column(index, title, nb_cards if nb_cards else None)
+
+    def edit_description(self):
+        description_editor = BoardDescription(self.get_description())
+        answer = self.modal.call(popin.Modal(description_editor))
+        if answer:
+            self.set_description(answer)
+
```

Comment:
I should be able to enter an empty description like it was before you change: remove the test.

```python
self.set_description(self.modal.call(popin.Modal(description_editor)))
```